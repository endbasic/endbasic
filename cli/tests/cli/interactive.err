endbasic: 20:1: Unknown builtin HELP
