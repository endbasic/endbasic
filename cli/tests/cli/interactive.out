
    RUN

    Runs the stored program.

    Note that the program runs in the context of the interpreter so it will
    pick up any variables and other state that may already be set.

